docs/vsphere_ci/README.md

```diff
@@ -0,0 +1,178 @@
+# Building Windows VM images in vSphere
+
+## Goals
+This document focuses on building Windows images in vSphere 6.7 environment used in CI but can be used as example for
+other vSphere environments. We use Packer to automate installation and configuration of the Windows images. Packer 
+generates a vSphere VM template that can be converted to a VM. After the VM has been created, the machine-api can clone
```

Comment:
I am confused about `Packer generates a vSphere VM template that can be converted to a VM. After the VM has been created,`. So we need a running VM?

There is a missing piece here. What gets plugged into the `MachineSet` needs to be called out.

Reply:
We have always tested with a Windows VM. Did not fully understand your question `So we need a running VM`.

Reply:
The `MachineSet` needs something in `providerSpec.templateName`. Does this point to a running VM or VM image (VMDK) or VM name?